DuckDuckGo/Statistics/GeneralPixel.swift

```diff
@@ -930,6 +934,15 @@ enum GeneralPixel: PixelKitEventV2 {
         case .contentBlockingCompilationTime:
             return "content_blocking_compilation_time"
 
+        case .contentBlockingLookupRulesSucceeded:
+            return "m_content_blocking_lookup_rules_succeeded"
+        case .contentBlockingFetchLRCSucceeded:
+            return "m_content_blocking_fetch_lrc_succeeded"
+        case .contentBlockingNoMatchInLRC:
+            return "m_content_blocking_no_match_in_lrc"
+        case .contentBlockingLRCMissing:
+            return "m_content_blocking_lrc_missing"
```

Comment:
We should not be prefixing these with 'm' on macOS, as it gets populated automatically.